daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/BlobLengthUnparser.scala

```diff
@@ -75,20 +75,9 @@ abstract class BlobUnparserBase(override val context: ElementRuntimeData)
         lengthInBits)
     }
 
-    // This adds two new buffered DataOutputStreams. The first is specific to a
-    // Blob, and contains all the logic for delivering the blob content to the
-    // current data output stream once it becomes direct without loading all
-    // the blob data into memory. The second is a buffered data output stream
-    // that Unparsers following this blob one will write data to. Note that if
-    // the current data stream is direct, the call to setFinished will cause
-    // Daffodil to immediately deliver the blob content to the direct output
-    // stream and make the blob data output stream the new direct. Since the
-    // blob data output stream is marked as finished in addBufferedBlob, the
-    // second data output stream will then be delivered, finally making it the
-    // direct stream.
```

Comment:
This comment is still valid and provides useful information about how blobs are unparsed. It might need to be tweaked a bit since there is no longer a special BLOB DOS, but i think this information is important.